web/styles/popovers.css

```diff
@@ -448,21 +458,18 @@ ul {
         }
     }
 
-    .user_profile_edit_button {
+    #user_profile_edit_button_icon {
```

Comment:
Is there a reason to change this to an ID instead of continuing to use a class?

Reply:
actually the `user_profile_edit_button_icon` element was wrapped by the element having class `user_profile_edit_button` and we were using the parent element to set the CSS properties, but with new changes we have removed the parent element and now only the child element having id is left, so I replaced here the class with the child element id, if you want I can change the id to class.

Reply:
Yeah, I think that would be the better route, especially since the element in question is just an icon (versus a bigger landmark kind of thing).

Reply:
@karlstolley Changed it, could you please review again, thanks :)